plugins/chat/spec/requests/chat/api/current_user_threads_spec.rb

```diff
@@ -29,4 +29,23 @@
       end
     end
   end
+
+  describe "#thread_count" do
+    describe "success" do
+      it "works" do
+        get "/chat/api/me/threads/count"
+
+        expect(response.status).to eq(200)
```

Comment:
We might want to check we have current thread count too here, no need to retest the whole service behind, just a basic happy path check.